frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -530,3 +538,19 @@ export const getIsNightMode = createSelector(
   [getDisplayTheme],
   theme => theme === "night",
 );
+
+export const getHasModelActionsEnabled = createSelector(
+  [getMetadata],
+  metadata => {
+    if (!metadata) {
+      return false;
+    }
+
+    const databases = metadata.databases as Record<DatabaseId, Database>;
```

Comment:
metadata.databaseList()

Reply:
@ranquild is databaseList somehow better? types are not complete and overall it's marked as a deprecated method

Reply:
It has the correct type. It's not `Database`

Reply:
but I'm fine leaving this as it

Reply:
changed, but types do not match, left ts-expect-error